subsquid/schema.graphql

```diff
@@ -66,4 +66,11 @@ type PabloPoolAsset @entity {
   blockNumber: BigInt!
   "Unix timestamp in ms"
   calculatedTimestamp: BigInt!
-}
\ No newline at end of file
+}
+
+type BondedFinanceBondOffer @entity {
+  "Total amount of purchased bonds"
```

Comment:
Need to have an `ID!` field for this entity

Reply:
It is autogenerated. On the migration files (`/db/migrations`) you can see the following:
```
await db.query(`CREATE TABLE "bonded_finance_bond_offer" ("id" character varying NOT NULL, "purchased" numeric NOT NULL, "asset_id" numeric NOT NULL, "beneficiary" text NOT NULL, CONSTRAINT "PK_1a7a97e3d57a4ac842dc2ef48ba" PRIMARY KEY ("id"))`)
```

Reply:
What is inserted as the ID when inserting data then?

Reply:
The `offerId`, which is unique.
```
new BondedFinanceBondOffer({
  id: offerId.toString(),
  purchased: BigInt(0),
  beneficiary,
})
```

Do you prefer a uuid instead?

Reply:
Maybe I'm confused but isn't the beneficiary the account that receives the assets?
```
pub struct BondOffer<AccountId, AssetId, Balance, BlockNumber> {
	/// The account that will receive the locked assets.
	pub beneficiary: AccountId,
	...
}
```

Reply:
If the need is only to track the beneficiary we need to make a change to the `NewOffer` event so that it contains the beneficiary field. Then we don't have a need to process `NewBond` event

Reply:
The need is to be able to query the total amount of purchased bonds for a given `offerId`, with `beneficiary` being a nice to have additional information

Reply:
We need to make sure that the entities here follow more or less the same relationships as the runtime so that there won't be a costly data migration every time FE requirements change.

Reply:
I mark this a `resolved` as it overlaps with the other conversation. Feel free to un-resolve it.